Formula/supervisor.rb

```diff
@@ -53,7 +63,7 @@ def plist
           <array>
             <string>#{opt_bin}/supervisord</string>
             <string>-c</string>
-            <string>#{etc}/supervisord.ini</string>
+            <string>#{etc}/supervisord.conf</string>
```

Comment:
Did you say this is no longer needed now?

Reply:
It's still needed for the plist because `supervisord` is started from `#{opt_bin}`, so it can't find the file according to its own heuristics: http://supervisord.org/configuration.html#configuration-file

Reply:
Gotcha 👍🏻